irods/test/modules/test_xml_parser.py

```diff
@@ -0,0 +1,2 @@
+import irods
+print(irods.client_configuration.connections.xml_parser_default)
```

Comment:
Let's leave a comment in this file referencing the test that uses it and why this is here for ease of reference in the future.

Reply:
done